lib/beaker-hostgenerator/data.rb

```diff
@@ -164,6 +164,15 @@ def osinfo
             'template' => 'amazon-7-arm64'
           }
         },
+        'archlinux5-64' => {
```

Comment:
I'm never really sure about the number. arch is rolling. facter determined the OS version by the kernel major version. newer facter version list the os major version as 'rolling'. I'm not sure if we can name it archlinux-64, but that might be a good idea.